pkgs/development/tools/rust/maturin/pyo3-test/generic.nix

```diff
@@ -0,0 +1,45 @@
+# Derivation prototype, used by maturin and setuptools-rust
+# passthrough tests.
+
+{ lib
+, fetchFromGitHub
+, python3Packages
+, rustPlatform
+
+, nativeBuildInputs
+
+, buildAndTestSubdir ? null
```

Comment:
```suggestion
, buildAndTestSubdir
```

Reply:
Then `generic.nix` wouldn't work from the `setuptools-rust` test, since it does not pass (or require) `buildAndTestDir`.